server/services/websockets.js

```diff
@@ -327,6 +327,22 @@ export default class Websockets {
         }
         return;
       }
+
+      case "fileOperations.update": {
+        return socketio
+          .to(`user-${event.actorId}`)
+          .emit("fileOperations.update", {
```

Comment:
Same question as before, this can use `presenters/fileOperation` can't it?

Reply:
I'm worried about passing the whole `user` object to the frontend.

Reply:
Should we emit `user` and `collection` objects? 

Observing the code in the file, my goal was to pass as little data as possible to the frontend.

Reply:
That's understandable, there are two reasons that we generally don't include the object in websocket messages –

1. scalability, when your team has 1000's of users that's just a lot more data over the wire
2. permissions – every team member has separate permissions and it's very easy to accidentally send them a websocket message with information they shouldn't have access to.

In this case with it only being sent down the single user channel I'm less wary of two cases, although the collection object in particular is still likely to be quite big. Which properties are we actually using at this point, is it just the `id` and `name` of each?

Reply:
We were using `name` and `url`. Because of the recent changes we only use `name` for now.

Reply:
In this scenario `collectionName` is totally okay rather than a partial `collection` object, sorry for the back and forth